src/Writers/Vipr.Writer.CSharp.Lite/FetcherSaveChangesAsyncMethod.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Dynamic;
+using Vipr.Core.CodeModel;
+
+namespace Vipr.Writer.CSharp.Lite
+{
+    public class FetcherSaveChangesAsyncMethod : Method
+    {
+        public OdcmClass OdcmClass { get; private set; }
+
+        private FetcherSaveChangesAsyncMethod(OdcmClass odcmClass)
+        {
+            Visibility = Visibility.Public;
+            IsOverriding = odcmClass.Base is OdcmClass && !((OdcmClass)odcmClass.Base).IsAbstract;
```

Comment:
Let's consider the IsAbstract check on SaveChanges, Update, Delete, Execute. Should these not exist, though the abstract type itself cannot be instatiated? this would allow iterating through collections of the base type to update/delete/save the derived types. This is likely a new feature.